specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json

```diff
@@ -0,0 +1,700 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-02-02-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/import": {
```

Comment:
This change is to make it more consistent since servers import and export are actions directly on the object. I am not sure about the history of this API and why we used the extensions path. 

Also, for the customers who are still using the old format of API, it will still work if they use the 2014 version, just not the 2020 version. So it should not be a problem AFAIK, because anyone using the new version will be writing new code. Let me know if that's not acceptable, and what is the usual way to provide the compatibility for these situations.

Reply:
Changing the URI like this breaks RBAC users have set up to give people access to import operations since it is now a new action that requires a different privilege.  Its up to your service and customers whether thats appropriate given the -preview nature of these APIs (although the 2014 API has been around for years)